src/imitation/scripts/multi_expert_demos.py

```diff
@@ -0,0 +1,150 @@
+"""Multiprocessed version of expert_demos."""
+from collections import OrderedDict
+import csv
+import multiprocessing.dummy
+import os.path as osp
+from typing import Iterable, List, Optional, Tuple
+
+from sacred.observers import FileStorageObserver
+import tensorflow as tf
+
+from imitation.scripts.config.multi_expert_demos import multi_expert_demos_ex
+from imitation.scripts.expert_demos import expert_demos_ex
+import imitation.util.multi as multi_util
+
+
+@multi_util.args_single_to_star
+def _job(
+  row: OrderedDict,
+  log_dir: str,
+  extra_named_configs: Iterable[str],
+  extra_config_updates: dict,
+) -> Tuple[OrderedDict, dict]:
+  """Start a `expert_demos_ex` run.
+
+  Params:
+    row: A CSV row encoding config settings for this run. Its fields should
```

Comment:
Avoid passing arguments inside a dictionary. This makes it harder to reason about function parameters, and loses a lot of the static type analysis. Instead make them top-level arguments and pass them as **row if necessary.